src/mainsite/time-frames.ts

```diff
@@ -6,22 +6,6 @@ export const timeFramesOld = ["m5", "h1", "d1", "d7", "d30", "all"] as const;
 /** @deprecated use TimeFrame instead, with an API map to convert to/from legacy time frames. */
 export type TimeFrameOld = typeof timeFramesOld[number];
 
-export const timeFramesNoMerge = [
```

Comment:
Amazing 🙌 .

Finally getting rid of those 👏 !

Reply:
Yes finally we can say "all timeframes are created equal"😅